swagger.yml

```diff
@@ -99,13 +108,18 @@ paths:
         '200':
           description: Successful request
           content:
+            # Codegen Workaround: codegen doesn't support multiple schemas. 
+            # Generic object has to be the first response type
+            application/pds4+xml:
+              schema:
+                type: object
             application/json:
               schema:
                 $ref: '#/components/schemas/product'
-            application/xml:
+            application/pds4+json:
```

Comment:
@tdddblog is there are reason that we do not have the `application/pds4+json` available for all of the endpoints (e.g. `bundles/{lidvid}/collections`? it seems like this should apply everywhere.

Reply:
@tdddblog ☝️

Reply:
@jordanpadams In the original story, only 3 endpoints were listed `/bundles/{lidvid}`, `/collections/{lidvid}`, and `/product/{lidvid}`. We also implemented `/bundles`, `/collections`, and `/products`. If we want this functionality for all endpoints we need to plan for it and estimate the effort based on total number of endpoints.